h/api/presenters.py

```diff
@@ -54,7 +54,8 @@ def tags(self):
 
     @property
     def target(self):
-        target = {'source': self.annotation.target_uri}
+        target = {'source': self.annotation.target_uri,
+                  'scope': self.annotation.target_uri_normalized}
```

Comment:
The celery tasks now get an id, which means we load the annotation from the database, so we have access to the `target_uri_normalized` property again, so all of this is not needed anymore.
Removed these 2-3 commits.